packages/integration-karma/test/native-shadow/style/style.spec.js

```diff
@@ -0,0 +1,30 @@
+import { createElement } from 'lwc';
```

Comment:
This should be covered by: `lwc/packages/integration-karma/test/shadow-dom/stylesheet/index.spec.js`